src/rez/solver.py

```diff
@@ -831,25 +838,21 @@ def sort_versions(self):
         The order is typically descending, but package order functions can
         change this.
         """
+        from rez.package_order import get_orderer
+
         if self.sorted:
             return
 
-        for orderer in (self.solver.package_orderers or []):
-            entries = orderer.reorder(self.entries, key=lambda x: x.package)
-            if entries is not None:
-                self.entries = entries
-                self.sorted = True
+        orderer = get_orderer(self.package_name, self.solver.package_orderers or {})
```

Comment:
```suggestion
        orderer = get_orderer(self.package_name, orderers=self.solver.package_orderers or {})
```
And same thing about `or {}`.